### PR TITLE
cloud: export console

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/serverless-export.md [hunk 1]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
+- If the storage space of stashing area is full, you will not be able to export data to the local file.
 
-**Amazon S3**
+### Amazon S3
 
-To export data to Amazon S3, you need to provide an [access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for your S3 bucket. Make sure the access key has read and write access for your S3 bucket, including at least these permissions: `s3:PutObject` and `s3:ListBucket`.
+To export data to Amazon S3, you need to provide the following information:
 
-### Data filtering
+- URI: `s3://<bucket-name>/<file-path>`
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:36]: 不确定结尾要不要加 * 号

File: tidb-cloud/serverless-external-storage.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+    3. Fill in the **Role Name**.
+
+    4. Acknowledge to create a new role and click **Create stack** to create the role ARN.
+
+    5. After the CloudFormation stack is executed, you can find the Role ARN value in the **Outputs** tab.
+   
+        ![img.png](/media/tidb-cloud/serverless-external-storage/serverless-role-arn.png)
+
+If you have any trouble creating a role ARN with AWS CloudFormation, you can take the following steps to create one manually:
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply tidb-cloud/serverless-external-storage.md:59]: Done

File: tidb-cloud/serverless-external-storage.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+It is recommended that you use [AWS CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) to create a role ARN. Take the following steps to create one:
+
+1. Open the **Import** page for your target cluster.
+
+    1. Log in to the [TiDB Cloud console](https://tidbcloud.com/) and navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+
+2. Open the **Add New ARN** dialog.
+
```
Comment (hfxsd) [inline tidb-cloud/serverless-external-storage.md:31]:
```suggestion
```

File: tidb-cloud/serverless-export.md [hunk 4]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+    - **Folder URI**: enter the URI of the Amazon S3 with the `s3://<bucket-name>/<file-path>` format.
+    - **Bucket Access**
+        - **AWS Role ARN**: enter the role ARN that has the permission to access the bucket.
+        - **AWS Access Key**: enter the access key ID and access key secret that have the permission to access the bucket.
 
-### Export with snappy compression
+4. Click **Export**.
+
+5. After the export task is successful, you can download the exported data to your local file with TiDB Cloud CLI.
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:150]: 对于 Amazon S3，不需要再单独通过 CLI 下载数据了吧
```suggestion
```

File: tidb-cloud/serverless-export.md [hunk 5]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
 
-**Local storage**
+### A local file
 
-Exporting data to local storage has the following limitations:
+Exporting data to a local file has the following limitations:
 
-- Exporting multiple databases to local storage at the same time is not supported.
+- You can only download the exported data using the [TiDB Cloud CLI](/tidb-cloud/cli-reference.md). Downloading using the TiDB Cloud console is not supported.
 - Exported data is saved in the stashing area and will expire after two days. You need to download the exported data in time.
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:29]:
```suggestion
- Exported data is saved in the stashing area of TiDB Cloud and will expire after two days. You need to download the exported data in time.
```

File: tidb-cloud/serverless-export.md [hunk 6]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
+
+4. Click **Export**.
+
+5. After the export task is successful, you can download the exported data to your local file using [TiDB Cloud CLI](/tidb-cloud/cli-reference.md).
+
+</div>
+
+<div label="CLI">
+
+1. Create an export task. TiDB Cloud CLI exports the data to a local file by default:
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/serverless-export.md:107]: reason: 其实这条命令只是导出到了一个 staging area

File: tidb-cloud/serverless-export.md [hunk 7]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
+    ```
 
     You will get an export ID from the output.
 
-2. After the export is successful, download the exported data to your local storage:
+2. After the export task is successful, download the exported data to your local file:
+
+    ```shell
+    ticloud serverless export download -c <cluster-id> -e <export-id>
+    ```
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:120]:
```suggestion

     For more information about the download command, see [ticloud serverless export download](/tidb-cloud/ticloud-serverless-export-download.md).
 
```

File: tidb-cloud/serverless-export.md [hunk 8]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
-```shell
-ticloud serverless export create -c <cluster-id> --database <database>
-```
+3. Configure the export information. On the **Import** page, click **Export Data to** in the upper-right corner, then choose **Amazon S3** from the drop-down list. Fill in the following parameters:
+
+    - **Task Name**: enter a name for the export task. The default value is `SNAPSHOT_{snapshot_time}`.
+    - **Exported Data**: choose the databases and tables you want to export.
+    - **Data Format**: choose **SQL File** or **CSV**.
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+    - **Folder URI**: enter the URI of the Amazon S3 with the `s3://<bucket-name>/<file-path>` format.
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:143]:
```suggestion
    - **Folder URI**: enter the URI of the Amazon S3 with the `s3://<bucket-name>/<folder-path>/` format.
```

File: tidb-cloud/serverless-external-storage.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+    - If you want to import data from Amazon S3, open the **Add New ARN** dialog as follows:
+
+        1. Click **Import from S3**.
+        2. Fill in the **File URI** field.
+        3. Choose **AWS Role ARN** and click **Click here to create new one with AWS CloudFormation**.
+        
+     - If you want to export data to Amazon S3, open the **Add New ARN** dialog as follows:
```
Comment (qiancai) [inline tidb-cloud/serverless-external-storage.md:38]:
```suggestion
    - If you want to export data to Amazon S3, open the **Add New ARN** dialog as follows:
```

File: tidb-cloud/serverless-export.md [hunk 10]
```diff
@@ -5,124 +5,195 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
-   ```shell
-   ticloud serverless export create -c <cluster-id> --database <database> --table <table>
-   ```
+3. On the **Import** page, click **Export Data to** in the upper-right corner, then choose **Local File** from the drop-down list. Fill in the following parameters:
+
+    - **Task Name**: enter a name for the export task. The default value is `SNAPSHOT_{snapshot_time}`.
+    - **Exported Data**: choose the databases and tables you want to export.
+    - **Data Format**: choose **SQL File** or **CSV**.
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+
```
Comment (hfxsd) [inline tidb-cloud/serverless-export.md:98]:
```suggestion

   > **Tip:**
   >
   > If you have not yet imported any data before, you need to click **Click here to export data to...** at the bottom of the page to export data.

```

File: tidb-cloud/serverless-export.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (shiyuhang0) [reply tidb-cloud/serverless-export.md:107]: 对的 ：）

File: tidb-cloud/serverless-export.md (hunk 4 above)
Comment (shiyuhang0) [reply tidb-cloud/serverless-export.md:150]: 不需要

File: tidb-cloud/serverless-export.md [hunk 11]
```diff
@@ -5,124 +5,199 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
-ticloud serverless export create -c <cluster-id> --database <database>
-```
+3. On the **Import** page, click **Export Data to** in the upper-right corner, then choose **Amazon S3** from the drop-down list. Fill in the following parameters:
+
+    - **Task Name**: enter a name for the export task. The default value is `SNAPSHOT_{snapshot_time}`.
+    - **Exported Data**: choose the databases and tables you want to export.
+    - **Data Format**: choose **SQL File** or **CSV**.
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+    - **Folder URI**: enter the URI of the Amazon S3 with the `s3://<bucket-name>/<folder-path>/` format.
+    - **Bucket Access**：choose one of the following access credentials and then fill in the credential information. If you do not have such information, see [Configure External Storage Access for TiDB Serverless](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
```
Comment (hfxsd) [inline tidb-cloud/serverless-export.md:148]:
```suggestion
    - **Bucket Access**: choose one of the following access credentials and then fill in the credential information. If you do not have such information, see [Configure External Storage Access for TiDB Serverless](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
```

**Review by hfxsd — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/serverless-export.md [hunk 12]
```diff
@@ -5,124 +5,203 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
+    - **Data Format**: choose **SQL File** or **CSV**.
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+    - **Folder URI**: enter the URI of the Amazon S3 with the `s3://<bucket-name>/<folder-path>/` format.
+    - **Bucket Access**: choose one of the following access credentials and then fill in the credential information. If you do not have such information, see [Configure External Storage Access for TiDB Serverless](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
+        - **AWS Role ARN**: enter the role ARN that has the `s3:PutObject` and `s3:ListBucket` permissions to access the bucket.
+        - **AWS Access Key**: enter the access key ID and access key secret that have the `s3:PutObject` and `s3:ListBucket` permissions to access the bucket.
 
-### Export with snappy compression
+4. Click **Export**.
+
```
Comment (hfxsd) [inline tidb-cloud/serverless-export.md:157]:
```suggestion
```

File: tidb-cloud/serverless-export.md [hunk 13]
```diff
@@ -5,124 +5,203 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
 
+- `s3.bucket-uri`: the Amazon S3 URI with the `s3://<bucket-name>/<file-path>` format.
+- `s3.access-key-id`: the access key ID of the user who has the permission to access the bucket.
+- `s3.secret-access-key`: the access key secret of the user who has the permission to access the bucket.
+
+</div>
+</SimpleTab>
+
 ### Cancel an export task
+To cancel an ongoing export task, take the following steps:
```
Comment (hfxsd) [inline tidb-cloud/serverless-export.md:175]:
```suggestion

To cancel an ongoing export task, take the following steps:
```

File: tidb-cloud/serverless-export.md [hunk 14]
```diff
@@ -5,124 +5,203 @@ summary: Learn how to export data from TiDB Serverless clusters.
...
-   ticloud serverless export create -c <cluster-id> --database <database> --table <table>
-   ```
+    - **Task Name**: enter a name for the export task. The default value is `SNAPSHOT_{snapshot_time}`.
+    - **Exported Data**: choose the databases and tables you want to export.
+    - **Data Format**: choose **SQL File** or **CSV**.
+    - **Compression**: choose **Gzip**, **Snappy**, **Zstd**, or **None**.
+
+   > **Tip:**
+   >
+   > If you have neither imported nor exported any data before, you need to click **Click here to export data to...** at the bottom of the page to export data.
```
Comment (qiancai) [inline tidb-cloud/serverless-export.md:103]:
```suggestion
   > If your cluster have neither imported nor exported any data before, you need to click **Click here to export data to...** at the bottom of the page to export data.
```

File: tidb-cloud/serverless-external-storage.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+
+    - If you want to import data from Amazon S3, open the **Add New ARN** dialog as follows:
+
+        1. Click **Import from S3**.
+        2. Fill in the **File URI** field.
+        3. Choose **AWS Role ARN** and click **Click here to create new one with AWS CloudFormation**.
+        
+    - If you want to export data to Amazon S3, open the **Add New ARN** dialog as follows:
+
+        1. Click **Click here to export data to**  > **Amazon S3**.
```
Comment (qiancai) [inline tidb-cloud/serverless-external-storage.md:40]:
```suggestion
        1. Click **Export data to...**  > **Amazon S3**. If your cluster has neither imported nor exported any data before, click **Click here to export data to...** > **Amazon S3** at the bottom of the page.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: media/tidb-cloud/serverless-external-storage/serverless-role-arn.png
Comment (qiancai) [inline media/tidb-cloud/serverless-external-storage/serverless-role-arn.png:1]: 这幅图里 Value 列里的值需要模糊处理下

Comment (zhangyangyu) [reply media/tidb-cloud/serverless-external-storage/serverless-role-arn.png:1]: 应该没关系，这种信息都不是隐私信息需要隐藏。而是靠本身的权限控制来保证安全。

**Pull request merged**